src/haxeLanguageServer/features/haxe/FindReferencesFeature.hx

```diff
@@ -34,12 +34,17 @@ class FindReferencesFeature {
 			offset: offset,
 			kind: WithBaseAndDescendants
 		}, token, locations -> {
-			resolve(locations.filter(location -> location != null).map(location -> {
-				{
-					uri: location.file.toUri(),
-					range: location.range
+			resolve([
+				for (location in locations) {
+					if (location == null)
```

Comment:
This looks extremely strange to me. :D 

![image](https://user-images.githubusercontent.com/2620907/216993989-3d966e45-2299-4bf9-92ec-a8addc9ba674.png)

We normally don't use bracket-less ifs in this codebase anyway.

Reply:
That's how formatter formatted it, though :thinking:

Reply:
I'm not sure there _is_ a better way to format this tbh, still looks strange.